tools/create-app.bash

```diff
@@ -24,10 +24,10 @@ cp $GOOGLE_SERVICES_PATH $DIRECTORY_PATH/../app/App_Resources/Android/
 # Parse json configuration file to string
 STRINGIFIED_CONFIGURATION=$(jq '. | tostring' $CONFIGURATION_FILE)
 
-APP_NAME=$(jq '.name | tostring' $CONFIGURATION_FILE | tr -d "\"")
+APP_ID=$(jq '.client[0].client_info.android_client_info.package_name | tostring' $GOOGLE_SERVICES_PATH | tr -d "\"")
```

Comment:
It seems `jq` is not a built in command in some linux operating systems. Did you happen to have done `sudo apt-get install jq`?

Reply:
I can add this dependency need to the script documentation

Reply:
> Did you happen to have done sudo apt-get install jq?

No, I didn't

> I can add this dependency need to the script documentation

great!